api-reference/50 Common/utils/excelExporter/exportDataGrid(options).md

```diff
@@ -197,6 +197,8 @@ You can call this method at any point in your application. In the example below,
 
 ---     
 
+[note]When the **DataGrid** is exported, the [detail rows](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/masterDetail/) are not exported. The [group rows](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Row/#rowType) are always exported expanded and the [isExpanded](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Row/#isExpanded) option is ignored.
```

Comment:
Let's make it a list of specificities. I'm sure there will be more.

Reply:
I could not find them after having a conversation with @EugeniyKiyashko a few days ago. May be @IgnatovDan can correct this?